src/main/java/decodes/tsdb/algo/ScalerAdder30.java

```diff
@@ -0,0 +1,290 @@
+	String _inputNames[] = { "input1", "input2", "input3", "input4", "input5", "input6", "input7", "input8", "input9", "input10", "input11", "input12", "input13", "input14", "input15", "input16", "input17", "input18", "input19", "input20", "input21", "input22", "input23", "input24", "input25", "input26", "input27", "input28", "input29", "input30" };
+//AW:INPUTS_END
+
+//AW:LOCALVARS
+
+//AW:LOCALVARS_END
+
+//AW:OUTPUTS
+	public NamedVariable output = new NamedVariable("output", 0);
+	String _outputNames[] = { "output" };
+//AW:OUTPUTS_END
+
+//AW:PROPERTIES
+	public double coeff1 = 1.0;
+	public double coeff10 = 1.0;
+	public double coeff11 = 1.0;
+	public double coeff12 = 1.0;
+	public double coeff13 = 1.0;
+	public double coeff14 = 1.0;
+	public double coeff15 = 1.0;
+	public double coeff16 = 1.0;
+	public double coeff17 = 1.0;
+	public double coeff18 = 1.0;
+	public double coeff19 = 1.0;
+	public double coeff2 = 1.0;
+	public double coeff20 = 1.0;
+	public double coeff21 = 1.0;
+	public double coeff22 = 1.0;
+	public double coeff23 = 1.0;
+	public double coeff24 = 1.0;
+	public double coeff25 = 1.0;
+	public double coeff26 = 1.0;
+	public double coeff27 = 1.0;
+	public double coeff28 = 1.0;
+	public double coeff29 = 1.0;
+	public double coeff3 = 1.0;
+	public double coeff30 = 1.0;
+	public double coeff4 = 1.0;
+	public double coeff5 = 1.0;
+	public double coeff6 = 1.0;
+	public double coeff7 = 1.0;
+	public double coeff8 = 1.0;
+	public double coeff9 = 1.0;
+	public String input10_MISSING = "ignore";
+	public String input11_MISSING = "ignore";
+	public String input12_MISSING = "ignore";
+	public String input13_MISSING = "ignore";
+	public String input14_MISSING = "ignore";
+	public String input15_MISSING = "ignore";
+	public String input16_MISSING = "ignore";
+	public String input17_MISSING = "ignore";
+	public String input18_MISSING = "ignore";
+	public String input19_MISSING = "ignore";
+	public String input1_MISSING = "ignore";
+	public String input20_MISSING = "ignore";
+	public String input21_MISSING = "ignore";
+	public String input22_MISSING = "ignore";
+	public String input23_MISSING = "ignore";
+	public String input24_MISSING = "ignore";
+	public String input25_MISSING = "ignore";
+	public String input26_MISSING = "ignore";
+	public String input27_MISSING = "ignore";
+	public String input28_MISSING = "ignore";
+	public String input29_MISSING = "ignore";
+	public String input2_MISSING = "ignore";
+	public String input3_MISSING = "ignore";
+    public String input30_MISSING = "ignore";
+	public String input4_MISSING = "ignore";
+	public String input5_MISSING = "ignore";
+	public String input6_MISSING = "ignore";
+	public String input7_MISSING = "ignore";
+	public String input8_MISSING = "ignore";
+	public String input9_MISSING = "ignore";
+	String _propertyNames[] = { "coeff1", 
+    "coeff10", "coeff11", "coeff12", "coeff13", "coeff14", "coeff15", "coeff16", "coeff17", "coeff18", "coeff19", "coeff2", 
```

Comment:
trying grouping these by 5s. And starting the data on the next line

```java
String _propertyNames[] =
{ 
    "coeff1", "coeff2", "coeff3", "coeff4", "coeff5",
    "coeff6", "coeff7", "coeff8", "coeff9", "coeef10"
// and so on
}
```

This will make it easier to verify everything is presented and accounted for.